dsgnwrks-instagram-importer.php

```diff
@@ -60,6 +60,13 @@ class DsgnWrksInstagram extends DsgnWrksInstagram_Debug {
 	 */
 	public $manual_import  = false;
 
+	/**
+	 * Plugin prefix
+	 *
+	 * @var string
+	 */
+	public $pre = 'dsgnwrks_instagram_';
```

Comment:
I didn't actually do this change; I think my wordpress installation must have a newer version of the plugin then what you have here on github (?)